apps/webpack.js

```diff
@@ -30,6 +30,7 @@ var baseConfig = {
       '@cdo/apps': path.resolve(__dirname, 'src'),
       '@cdo/static': path.resolve(__dirname, 'static'),
       repl: path.resolve(__dirname, 'src/noop'),
+      http: 'stream-http',
```

Comment:
Yes. Mostly I'd just like to see it documented somewhere why this particular config is necessary.